### DISCUSSION TITLE
formily_next 分支 为什么不需要 bootstrap 指令？

### DISCUSSION THREAD

**Discussion by WuHuSa** [alibaba/formily#2801] [Q&A 2.x] [answered]
源码中，master 分支内 scripts 中有 bootstrap 指令给所有子包安装依赖，
而 formily_next 分支中 scripts 不再有 bootstrap 指令，
诚心求教，是通过什么方式处理的

**Comment by janryWang** [comment, marked as answer]
用的yarn
